prunning_implementation/prunning.py

```diff
@@ -0,0 +1,99 @@
+import babeltrace
+from babeltrace import CTFWriter
+import sys
+import tempfile
+from collections import Counter
+from collections import deque
+
+
+#############################################################################
+#    Function to Prunn the data and put in a deque                          #
+#    @param trace_path: The path to the stream file                #
+#           capacity: The capacity of the deque                             #
+############################################################################
```

Comment:
I would prefer we should use Google's Python style, and to use pydoc comments for this kind of documentation. As a reference, have a look at the first few functions in the ../J1939_experiments/experiment.py file.

Reply:
https://google.github.io/styleguide/pyguide.html